jvm/src/vyxal/gen/GenerateDocs.scala

```diff
@@ -57,4 +59,46 @@ private object GenerateDocs:
     SugarMap.trigraphs
       .map { case (key, value) => s"$key -> $value" }
       .mkString("", "\n", "\n")
+
+  def elementTable(): String =
+    val header = "| Symbol | Name | Keywords | Arity | Vectorises | Overloads |"
+    val divider = "| --- | --- | --- | --- | --- | --- |"
+    val contents = StringBuilder()
+    val formatOverload = (overload: String) =>
```

Comment:
```suggestion
    def formatOverload(overload: String) =
```
Makes the intent more clear (I thought at first that you needed to pass `formatOverload` elsewhere). If you want to use it like a helper, it's better to make it a normal nested method rather than making a function object